net/ostiary/Makefile

```diff
@@ -0,0 +1,50 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=ostiary
+PKG_VERSION:=4.0
+PKG_RELEASE:=1
+
+PKG_LICENSE:=GPL-2.0-only
```

Comment:
PKG_MAINTAINER above PKG_LICENSE and placed one line below PKG_CAT.

Reply:
Re-ordered/moved